src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/leaked-passwords/LeakedPasswordsLayout.tsx

```diff
@@ -174,8 +175,9 @@ export function LeakedPasswordsLayout(props: LeakedPasswordsLayoutProps) {
           ? "password_exposed"
           : "security_question_exposed",
       utm_source: "guided_experience",
+      plan_tier: props.isPremium ? "Plus" : "Free",
```

Comment:
Should the `"Plus"` and `"Free"` be in an enum somewhere?
Looks like Ctrl+F of "Plus" gives 6 hardcoded results and "Free" about 9.

Reply:
Thanks, enum is a good idea.

Reply:
I wish that we could define this in Glean and reference it there... AFAICT from the docs, keys on `Event`s are always `string` types so I don't think there's a great way to specify it.

Reply:
I think this is a good idea, but the way we need to pass down props to all of these components is temporary and I don't expect to keep this forever. I'd rather not have to import the type just for this in all of these. It might make sense to use a function instead of a boolean or something, but I think passing simpler prop types than function is generally preferred. I'd like to keep it as-is for now